src/web/src/contexts/ContextHelper.ts

```diff
@@ -0,0 +1,10 @@
+import { Context, useContext } from "react";
+
+export const useOptionalContext = <T>(ctx: Context<T | undefined>): T => {
```

Comment:
Is this one used?

Reply:
Hmm a code-smell I suspect. I am creating the context without initial state, and this results to a nullable context. This specific helper is used a lot.